<launch>
    <arg name="sys" default="false"/>
    <group if="$(arg sys)">
        <include file="$(find dbw_polaris_can)/launch/dbw.launch" />
    </group>

    <arg name="brake_gain" default=".3"/>
    <arg name="throttle_gain" default=".2"/>
    <node ns="vehicle" pkg="joystick_demo2" type="joystick_demo2" name="joystick_demo2" output="screen" >
        <param name="brake_gain" value="$(arg brake_gain)" /> <!-- Adjust brake value -->
        <param name="throttle_gain" value="$(arg throttle_gain)" /> <!-- Adjust throttle value -->
    </node>


    <node name = "joy_node" pkg = "joy" type = "joy_node"/>
    <!--<node name = "joy_node" pkg = "joy" type = "joy_node" output = "screen" launch-prefix = "gnome-terminal -x" />-->
    <!--<param name = "autorepeat_rate" value="5"/>-->
    
    
    <!--<node name = "TPcontrol_joy_node" pkg = "prizm-control-hw-Thomasbrefeld" type = "joy_control_node"/>-->
    <node name = "joy_to_twist" pkg = "joystick_demo2" type = "joy_to_twist" output = "screen" launch-prefix = "gnome-terminal -x" />
</launch>